p2p/transport/websocket/websocket.go

```diff
@@ -186,6 +187,12 @@ func (t *WebsocketTransport) maDial(ctx context.Context, raddr ma.Multiaddr) (ma
 			copytlsClientConf := t.tlsClientConf.Clone()
 			copytlsClientConf.ServerName = sni
 			dialer.TLSClientConfig = copytlsClientConf
+			ipAddr := wsurl.Host
+			dialer.NetDial = func(network, address string) (net.Conn, error) {
```

Comment:
Why is this needed? We're dealing with (DNS-) resolved addresses here anyway, right?

Reply:
Added a comment